scripts/postData.sh

```diff
@@ -1,33 +1,35 @@
 #!/bin/bash
-source $ALLSKY_HOME/config.sh
-source $ALLSKY_HOME/scripts/ftp-settings.sh
 
 # TODO Needs fixing when civil twilight happens after midnight
-cd $ALLSKY_HOME/scripts
-ME="$(basename "$BASH_ARGV0")"	# Include script name in output so it's easier to find in the log file
 
-latitude=$(jq -r '.latitude' "$CAMERA_SETTINGS")
-longitude=$(jq -r '.longitude' "$CAMERA_SETTINGS")
+source "${ALLSKY_HOME}/variables.sh"
+source "${ALLSKY_CONFIG}/config.sh"
+source "${ALLSKY_SCRIPTS}/ftp-settings.sh"
+
+ME="$(basename "${BASH_ARGV0}")"
+
+latitude=$(jq -r '.latitude' "${CAMERA_SETTINGS}")
+longitude=$(jq -r '.longitude' "${CAMERA_SETTINGS}")
 timezone=-0700
```

Comment:
This timezone probably isn't correct for everyone :)

`readlink /etc/localtime | sed -e 's,/usr/share/zoneinfo/,,'` can fetch the name of the timezone
`date "+%z"` and `date "+%:z"` can fetch the timezone in a numeric format